linkml/utils/generator.py

```diff
@@ -190,6 +193,13 @@ def visit_subset(self, subset: SubsetDefinition) -> None:
         """
         ...
 
+    def visit_enum(self, enum: EnumDefinition) -> None:
```

Comment:
@hsolbrig - this looks good to me but as this is changing the base class you may want to comment